cogs/ladder.py

```diff
@@ -14,17 +11,18 @@
 race_dict = {"P": "Protoss", "T": "Terran", "Z": "Zerg", "R": "Random"}
 
 
-class Ladder(commands.Cog, name="urls"):
+class Ladder(commands.Cog, name="ladder"):
```

Comment:
The name was otherweise a duplicate and did not start for me locally. I'm not sure how the discord bot currently runs at all.